hatchery/src/error.rs

```diff
@@ -25,6 +27,7 @@ pub enum Error {
     MissingModuleExport,
     CompositeSerializerError(Compo),
     PersistenceError(io::Error),
+    OutOfPoints(ModuleId),
```

Comment:
I don't see the reason to include the ModuleId here

Reply:
The user might appreciate knowing which contract was given too little points to operate.

Reply:
This can already be more elegantly deducted from what's on top of the call stack

Reply:
The call stack is currently not exposed to the caller

Reply:
it feels off to me, but not a big deal.